### PR TITLE
Add Thomas Vellekoop as a CIP editor

### PR DESCRIPTION
**Pull request by perturbing**
With this PR I would like to put myself forward in the role as a CIP editor. Given this [PR](https://github.com/cardano-foundation/CIPs/pull/901), I would love to fill these big shoes and contribute to the continuation of the CIP process as it is now. 

A big thanks to both @KtorZ and @SebastienGllmt for their effort and, above all, their great technical minds.

Regarding my competence for this role, I have contributed to various CIP's and even co-authored some myself. I hope others recognize this and have faith in my ability to execute this role successfully :smiley:

### CODE REVIEW

**Comment by Hornan7** [comment]
I vouch for this guy, for what its worth. 😇😅

**Review by Crypto2099 — APPROVED** [review]
Thomas would be a wonderful addition to the CIP Editors panel IMO and I'm happy to approve and endorse his nomination.

**Review by Ryun1 — APPROVED** [review]
I would second this nomination
Thomas is not only incredibly knowledgeable about Cardano but also has co-authored some great proposals
whilst continually contributing to the CIP process 💪

(For completeness lets wait for @rphair's review before merging)

**Pull request merged**
